Week3/homework/calculateTotalPrice.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const calculateTotalPrice = (object) => {
+    let sum = null;
```

Comment:
nit: might be clearer to do `let sum = 0;`

Reply:
Thank you for the feedback I fixed